### PR TITLE
AM-449- Run application on container.

### PR DESCRIPTION
**Pull request by dharmendra8186**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-497

### Change description ###
AM-449 Case Document AM API-Download: Implement 'Fetch metadata from Doc-store' API at case-document

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/StoredDocumentHalResource.java [hunk 1]
```diff
@@ -6,21 +6,29 @@
 import java.util.List;
```
Comment (aashishaggarwal123) [inline src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/StoredDocumentHalResource.java]: Can we replace all getters, setters and other constructor methods with Lombok annotation?

Comment (dharmendra8186) [reply src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/StoredDocumentHalResource.java]: I have done.

File: docker-compose.yml [hunk 2]
```diff
@@ -16,13 +16,14 @@ services:
       S2S_KEY: S2S_KEY
       CCD_DOCUMENT_API_IDAM_KEY:
       USER_PROFILE_HOST:
+      Document_STORE_URL: http://ccd-api-gateway:3453/documents
       AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY:
       IDAM_USER_URL: http://idam-api:5000
       IDAM_S2S_URL: http://service-auth-provider-api:8080
-      #JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+      JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
```
Comment (nitishsingla1234) [inline docker-compose.yml]: We should keep the Java tool options disabled and let the container decide the memory profile for the application

Comment (nitishsingla1234) [reply docker-compose.yml]: It won't impact the container, hence keeping as is.

Comment (dharmendra8186) [reply docker-compose.yml]: It's doesn't affect on any functionality.

**Pull request merged**